### DISCUSSION TITLE
I can't update my GTS 4 watch

### DISCUSSION THREAD

**Discussion by mimimix** [zepp-health/zeppos-forum#369] [Q&A] [answered]
Recently saw that system version 3.5 has already been released for the gts 4. However, I have not received any update notifications. I went into the app to manually check for the update. But there is nothing there either.

What can I do about it?

P.s. The watch screen is intact, it's a protective film

![Screenshot_2024-09-19-09-27-58-427_com.yandex.browser.png](https://github.com/user-attachments/assets/c91d6147-0b05-49b1-b979-a347bcbe8eaa)

![Screenshot_2024-09-19-09-26-47-702_com.huami.watch.hmwatchmanager-edit.jpg](https://github.com/user-attachments/assets/9f1d2ae0-2055-4707-8b7f-e7205707a2f8)

![IMG_20240919_093301.jpg](https://github.com/user-attachments/assets/3acafdf7-4ceb-4575-8e4c-0651c85092ef)

**Comment by silver-zepp** [comment]
Update your Zepp app in order to receive a firmware update.

**Reply by mimimix** [reply]
I don't see any updates in the play market. The current version of zepp app for me: 8.12.5-play
@silver-zepp

**Reply by MyoungXUE** [reply, marked as answer]
@mimimix 
Please be patient and wait for a while. Currently, only a part of the Zepp OS 3.5 upgrade for GTS4 has been pushed.
